pkg/report/linux_test.go

```diff
@@ -511,3 +512,16 @@ func testDisassembly(t *testing.T, reporter *Reporter, linuxReporter *linux, tes
 		t.Fatalf("expected:\n%s\ngot:\n%s", output, result)
 	}
 }
+
+func TestFalseReboot(t *testing.T) {
```

Comment:
Test inputs for negative cases (no crash)  should start with empty line. There is no need to add a new Go test.